Entities/Structures/Trampoline/TrampolineLogic.as

```diff
@@ -62,6 +62,10 @@ void onCollision(CBlob@ this, CBlob@ blob, bool solid, Vec2f normal, Vec2f point
 	//cant bounce while held by something attached to something else
 	if (holder !is null && holder.isAttached()) return;
 
+	//prevent knights from flying using trampolines
+	if (blob.getPosition().y > this.getPosition().y && this.getAngleDegrees() == 0)
```

Comment:
Unless I'm reading this incorrectly, doesn't that mean that this also applies to any horizontal trampoline?

Reply:
That’s what the angle check is for.

Reply:
I think @AsuMagic 's concern is that this would prevent bouncing on horizontal tramps - it's when "bouncing" from below though, right? That will break some tramp "elevators" on TDM levels.

Could be better to require an upwards velocity rather than being above the tramp as a workaround that preserves the fix?